jina/orchestrate/deployments/config/k8slib/kubernetes_deployment.py

```diff
@@ -88,19 +88,50 @@ def get_template_yamls(
         'shard_id': f'\"{shard_id}\"' if shard_id is not None else '\"\"',
         'pod_type': pod_type,
         'protocol': str(protocol).lower() if protocol is not None else '',
-        'volume_path': volumes[0] if volumes is not None else None
+        'volume_path': volumes[0] if volumes is not None else None,
     }
 
     if gpus:
         template_params['device_plugins'] = {'nvidia.com/gpu': gpus}
 
     template_name = 'deployment-executor' if name != 'gateway' else 'deployment-gateway'
+    service_ports_section = '''
```

Comment:
I prefer to  keep this in `yaml`. This gives a single source of truth, and can be more easily changed

Reply:
You can have this in YAML, but then load and edit

Reply:
will put only this section as a separate file in resources

Reply:
done